extensions/gdx-setup/src/com/badlogic/gdx/setup/data/extensions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+    <extension>
+       <name>Overlap2D</name>
+       <description>level and ui editor runtime</description>
+       <package>com.underwaterapps.overlap2druntime</package>
+       <version>0.0.8</version>
+       <dependencies>
+           <core>com.underwaterapps.overlap2druntime:overlap2d-runtime-libgdx</core>
+       </dependencies>
+    </extension>
+</extensions>
```

Comment:
An adapted xml to support multiple dependencies per platform and unsupported platforms would look something like: 

```xml
<?xml version="1.0" encoding="UTF-8"?>
<extensions>
    <extension>
       <name>Overlap2D</name>
       <description>level and ui editor runtime</description>
       <package>com.underwaterapps.overlap2druntime</package>
       <version>0.0.8</version>
       <projects>
           <core>
               <dependency>com.underwaterapps.overlap2druntime:overlap2d-runtime-libgdx</dependency>
           </core>
           <desktop></desktop> //No additional dependencies 
           <android></android>  //No additional dependencies
           <ios>null</ios>  // Not supported
           <html>null</html>  //Not supported
       </projects>
    </extension>
</extensions>
```